demos/python_demos/human_pose_estimation_demo/human_pose_estimation_demo.py

```diff
@@ -0,0 +1,293 @@
+    io_args.add_argument('-u', '--utilization_monitors', default='', type=str,
+                         help='Optional. List of monitors to show initially.')
+
+    debug_args = parser.add_argument_group('Debug options')
+    debug_args.add_argument('-r', '--raw_output_message', help='Optional. Output inference results raw values showing.',
+                            default=False, action='store_true')
+    return parser
+
+
+def get_model(ie, args, aspect_ratio):
+    if args.architecture_type == 'ae':
+        Hpe = models.HpeAssociativeEmbedding
+    elif args.architecture_type == 'openpose':
+        Hpe = models.OpenPose
+    else:
+        raise RuntimeError('No model type or invalid model type (-at) provided: {}'.format(args.architecture_type))
+    return Hpe(ie, args.model, target_size=args.tsize, aspect_ratio=aspect_ratio)
+
+
+def put_highlighted_text(frame, message, position, font_face, font_scale, color, thickness):
+    cv2.putText(frame, message, position, font_face, font_scale, (255, 255, 255), thickness + 1)  # white border
+    cv2.putText(frame, message, position, font_face, font_scale, color, thickness)
+
+
+def get_plugin_configs(device, num_streams, num_threads):
+    config_user_specified = {}
+
+    devices_nstreams = {}
+    if num_streams:
+        devices_nstreams = {device: num_streams for device in ['CPU', 'GPU'] if device in device} \
+            if num_streams.isdigit() \
+            else dict(device.split(':', 1) for device in num_streams.split(','))
+
+    if 'CPU' in device:
+        if num_threads is not None:
+            config_user_specified['CPU_THREADS_NUM'] = str(num_threads)
+        if 'CPU' in devices_nstreams:
+            config_user_specified['CPU_THROUGHPUT_STREAMS'] = devices_nstreams['CPU'] \
+                if int(devices_nstreams['CPU']) > 0 \
+                else 'CPU_THROUGHPUT_AUTO'
+
+    if 'GPU' in device:
+        if 'GPU' in devices_nstreams:
+            config_user_specified['GPU_THROUGHPUT_STREAMS'] = devices_nstreams['GPU'] \
+                if int(devices_nstreams['GPU']) > 0 \
+                else 'GPU_THROUGHPUT_AUTO'
+
+    return config_user_specified
+
+
+default_skeleton = ((15, 13), (13, 11), (16, 14), (14, 12), (11, 12), (5, 11), (6, 12), (5, 6),
+    (5, 7), (6, 8), (7, 9), (8, 10), (1, 2), (0, 1), (0, 2), (1, 3), (2, 4), (3, 5), (4, 6))
+
+colors = (
+        (255, 0, 0), (255, 0, 255), (170, 0, 255), (255, 0, 85),
+        (255, 0, 170), (85, 255, 0), (255, 170, 0), (0, 255, 0),
+        (255, 255, 0), (0, 255, 85), (170, 255, 0), (0, 85, 255),
+        (0, 255, 170), (0, 0, 255), (0, 255, 255), (85, 0, 255),
+        (0, 170, 255))
+
+
+def draw_poses(img, poses, scores, pose_score_threshold, point_score_threshold, skeleton=default_skeleton, draw_ellipses=False):
+    if poses.size == 0:
+        return img
+    stick_width = 4
+
+    img_limbs = np.copy(img)
+    for pose, pose_score in zip(poses, scores):
+        if pose_score <= pose_score_threshold:
```

Comment:
The `pose_score` is always > 1 and condition is not satisfied.

Reply:
Updated